src/test/unit_tests/coretest.js

```diff
@@ -339,8 +340,9 @@ async function init_test_pools(client, system_name, pools_to_create) {
 
 // delete all test pools (including hosts and agents)
 async function clear_test_pools() {
+    console.log('CLEANING ALL CHUNKS');
+    await MDStore.instance().delete_all_chunks_in_system();
```

Comment:
I think this needs a comment that explains why we had to add it here.
It's not so easy to figure it out.